site/source/tutorials/007_theming/index.md

```diff
@@ -0,0 +1,136 @@
+
+{% instruction 'Create a new css file for `Banner` named `banner.m.css`.' %}
+
+We will create an empty `root` class for now as our base theme does not require any styles to be added to the `banner`.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/styles/banner.m.css' lang:css %}
+
+{% instruction 'Now, lets look at changing `WorkerForm`' %}
+
+`WorkerForm` already uses the `ThemeableMixin` and has a `workerForm` class on it's root node. Lets go ahead and change that to a `root` class, but while we're there, we will create a `rootFixed` class too and apply that as a `fixed` class to the root node. This passes a classname to the theming system that cannot be changed or overridden via a theme and ensures that structured or nested styles are not lost when a theme is used.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:48-51 %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/styles/workerForm.m.css' lines:1-17 lang:css %}
+
+If you open the application in a webpage it should still look and work the same as before.
+
+{% instruction 'Why don\'t you go ahead and make the same changes to `Worker` and `WorkerContainer`' %}
+
+Next, we'll start to create a theme.
+
+{% section %}
+
+## Creating a Theme
+
+{% task 'Create a theme directory' %}
+
+Lets go ahead and create a `theme` directory under your project `src`. This is where we will store our themes. For the purpose of this tutorial, we will create a `dojo` theme. Create a `theme.ts` under `src/themes/dojo/theme.ts`. This is the `theme` file into which we will import our themed css files and export the keyed theme for consumption by our widgets.
+
+{% instruction 'Theme the worker widget' %}
+
+In order to theme the `worker` widget, we will need to create a `worker.m.css` file within our theme directory and and use it within our `theme.ts` file. The naming here is important as the object key of the exported theme must match the naming of the widget's css file.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/theme.ts' lines:3,9,11,16 %}
+
+Lets start by creating a red `worker` so that we can see our theme has been applied correctly.
+
+```css
+/* worker.m.css */
+.root {
+	background: red;
+}
+```
+
+In order to apply our theme, we will need to use a `themeInjector`. Alternatively we *could* set `theme` on the top level widget and pass it down via properties to every child widget but this approach is not very elegant. We can create a `themeInjector` using the `registerThemeInjector` function exported by `ThemeableMixin`. So let's go ahead and update our `main.ts` file to import our theme and create a `themeInjector`.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/main.ts' %}
+
+If you open up your web browser now you should see that the `worker` backgrounds are now `red`.
```

Comment:
Open the application in your web browser and see that the `worker` backgrounds are now `red`.